ClLike/cl_like/limber.py

```diff
@@ -92,9 +94,11 @@ def _get_tracers(self, cosmo, **pars):
                 z = dndz[0]
                 oz = np.ones_like(z)
 
-                t0 = None
                 tr = ccl.NumberCountsTracer(cosmo, dndz=dndz, bias=(z, oz),
                                             has_rsd=False)
+                t0 = None
```

Comment:
Can you add a comment here saying what t0, t1, tn, etc are? I know now what they are (t0 == unbiased tracer, right?). But we might forget it in a few months

Reply:
Done